quinn-proto/src/connection/mod.rs

```diff
@@ -3222,8 +3255,33 @@ impl Connection {
             self.datagrams.send_blocked = false;
         }
 
-        // STREAM
         if space_id == SpaceId::Data {
+            // NEW_TOKEN
+            while self.path.new_tokens_to_send > 0 {
```

Comment:
I don't think it's predictable, because the user's configured `HandshakeTokenKey` and by extension `AeadKey` implementation may not add an amount of padding that we can predict.